src/features/pages/markets/etf/instruments-etf/styles.module.scss

```diff
@@ -2,6 +2,6 @@
 
 .item_container {
     @include breakpoints(tablet) {
-        padding: 2.4rem 22rem;
+        padding: 0 22rem;
```

Comment:
can we remove it all while we already have the same in `src/features/pages/markets/etf/instrumental-etf-content.tsx/styles.module.scss`?